app/infrastructure/services.py

```diff
@@ -63,13 +80,17 @@ def __init__(self, search_client: ElasticClient):
     def feed(self, reuse: Reuse) -> None:
         self.search_client.index_reuse(reuse)
 
-    def search(self, search_text: str, page: int, page_size: int) -> Tuple[List[Reuse], int, int]:
+    def search(self, args: dict) -> Tuple[List[Reuse], int, int]:
+        page = args.pop('page')
```

Comment:
`args` is of type ReuseArgs and not `dict` here

Reply:
My mistake, I forgot to the `.dict()` before. The received argument will be of type `dict` :)